colibre/scripts/dust_mass_evolution.py

```diff
@@ -0,0 +1,110 @@
+"""
+Plots the dust mass density evolution.
+"""
+import matplotlib
+
+import unyt
+
+import matplotlib.pyplot as plt
+import numpy as np
+import sys
+import glob
+
+
+from swiftsimio import load, load_statistics
+
+from swiftpipeline.argumentparser import ScriptArgumentParser
+
+from velociraptor.observations import load_observation
```

Comment:
```suggestion
from velociraptor.observations import load_observations
```

- `load_observation` is outdated. 
- Note also the associated suggestion below in this file where I use `load_observations`